utils/tests_fetcher.py

```diff
@@ -620,20 +627,29 @@ def infer_tests_to_run(output_file, diff_with_last_commit=False, filters=None, j
                 json.dump(test_map, fp, ensure_ascii=False)
 
 
-def filter_pipeline_tests(output_file):
+def filter_tests(output_file, filters):
+    """
+    Reads the content of the output file and filters out all the tests in a list of given folders.
+
+    Args:
+        output_file (`str` or `os.PathLike`): The path to the output file of the tests fetcher.
+        filters (`List[str]`): A list of folders to filter.
+    """
     if not os.path.isfile(output_file):
         print("No test file found.")
         return
     with open(output_file, "r", encoding="utf-8") as f:
         test_files = f.read().split(" ")
 
-    if len(test_files) == 0:
+    if len(test_files) == 0 or test_files == [""]:
         print("No tests to filter.")
         return
+
+    print(test_files)
```

Comment:
Sorry @sgugger I forgot to add a comment here. I feel this is also a debugging line and you intended to remove it. Just a nit and your call.

Reply:
Yes, this can be removed.